frontend/src/components/eMIB/Email.jsx

```diff
@@ -43,8 +46,10 @@ const styles = {
 class Email extends Component {
   static propTypes = {
```

Comment:
Can you add here a comment saying which props are provided by Redux?
The same way we added comments for other components that use Redux.